src/Commands/AutoTurn.cpp

```diff
@@ -0,0 +1,67 @@
+#include "AutoTurn.h"
+#include "../Subsystems/Chassis.h"
+
+AutoTurn::AutoTurn(TurnSide side, double distance)
+: mSide(side),mDistance(distance ){
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(Robot::chassis.get());
+  Requires(Chassis::getInstance().get());
+}
+
+// Called just before this Command runs the first time
+void AutoTurn::Initialize() {
+  Chassis::getInstance()->ConfigPeakOutput(8);
+  Chassis::getInstance()->SetupPID();
+  Chassis::getInstance()->ZeroPosition();
+  Chassis::getInstance()->SetShifter(Chassis::kShifterLow);
+  switch(mSide)
```

Comment:
I chose to use `switch/case` instead of `if/then/else`. For items like this, `case` seems more readable.